app/1.3.x/db-less-admin-api.md

```diff
@@ -1457,15 +1457,15 @@ HTTP 200 OK
 
 ---
 
-### Retrieve Ca_certificate
+### Retrieve Ca Certificate
```

Comment:
You can add `ca` to the `title_exceptions` array in `autodoc-admin-api/run.lua` so it is properly auto-capitalized :)

Reply:
Already pushed a commit to fix it.

Reply:
what I mentioned above is different from the `entity_title` entry

Reply:
Got it. That has been added.